### PR TITLE
Name tag exposed directly from AtlasEntity

### PR DESCRIPTION
**Pull request by yazad3**
### Description:
Exposed the standard "name" tag as a method in AtlasEntity. This does not address names in other languages currently.

### Potential Impact:
None.

### Unit Test Approach:
Unit Test added to check presence and absence of name.

### Test Results:
Successful execution of tests.

------

In doubt: [Contributing Guidelines](CONTRIBUTING.md)

### CODE REVIEW

File: src/main/java/org/openstreetmap/atlas/geography/atlas/items/AtlasEntity.java [hunk 1]
```diff
@@ -61,6 +61,16 @@ public boolean containsKey(final Iterable<String> matches)
...
     }
 
+    /**
+     * The value in the "name" attribute.
+     *
+     * @return an optional string representing the value of the name tag.
+     */
+    public Optional<String> getName()
+    {
+        return this.getTag("name");
```
Comment (matthieun) [inline src/main/java/org/openstreetmap/atlas/geography/atlas/items/AtlasEntity.java]: Can we use the NameTag's KEY here?

File: src/test/java/org/openstreetmap/atlas/geography/atlas/items/AtlasEntityTest.java [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+{
+
+    @Rule
+    public final AreaEntityTestRule rule = new AreaEntityTestRule();
+
+    @Test
+    public void getName()
+    {
+        Assert.assertEquals("abc", rule.getAtlas().node(1).getName().get());
+        Assert.assertTrue(!rule.getAtlas().node(2).getName().isPresent());
```
Comment (matthieun) [inline src/test/java/org/openstreetmap/atlas/geography/atlas/items/AtlasEntityTest.java]: AssertFalse instead, no?

File: src/main/java/org/openstreetmap/atlas/geography/atlas/items/AtlasEntity.java (hunk 1 above)
Comment (yazad3) [reply src/main/java/org/openstreetmap/atlas/geography/atlas/items/AtlasEntity.java]: Good suggestion!

File: src/test/java/org/openstreetmap/atlas/geography/atlas/items/AtlasEntityTest.java (hunk 2 above)
Comment (yazad3) [reply src/test/java/org/openstreetmap/atlas/geography/atlas/items/AtlasEntityTest.java]: done

**Pull request merged**